.idea/.name

```diff
@@ -0,0 +1 @@
+subject.py
```

Comment:
We should add `.idea/` to `.gitignore` as these are files generated by your IDE.